lib/matplotlib/text.py

```diff
@@ -324,14 +324,23 @@ def _get_layout(self, renderer):
             whs[i] = w, h
 
             baseline = (h - d) - thisy
-            thisy -= max(offsety, (h - d) * self._linespacing)
+            if i == 0:
+                # position at baseline
+                thisy = -(h - d)
+            else:
+                # put baseline a good distance from bottom of previous line
+                thisy -= max(offsety, (h - d) * self._linespacing)
             horizLayout[i] = thisx, thisy, w, h
             thisy -= d
             width = max(width, w)
             descent = d
 
-        ymin = horizLayout[-1][1]
-        ymax = horizLayout[0][1] + horizLayout[0][3]
+        # Bounding box definition:
+        # by defeinition, ymax is 0
+        ymax = 0
+        # ymin is the baseline of the last line subtract the descent of the
+        # last line.
```

Comment:
I don't understand the comment; can you condense to one line, and clarify?

Reply:
Reworded, though I admit that I don't completely remember what this meant.  This code is very convoluted, and could maybe do with a refactor now that we know what its supposed to do. ...